src/app/mission/[id]/stopwatch/page.tsx

```diff
@@ -80,8 +80,12 @@ export default function StopwatchPage() {
       setIsMoveLoading(true);
       removeProgressMissionData();
     },
-    onError: () => {
-      setIsMoveLoading(() => false); // 없어도 되는지 확인 필요
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    onError: (error: any) => {
+      if (error.response.data.data.errorClassName === 'MISSION_RECORD_ALREADY_EXISTS_TODAY') {
+        removeProgressMissionData();
+        router.replace(ROUTER.HOME);
+      }
     },
```

Comment:
isSeverError util 을 쓰면 sever 데이터 타입 추론되게 할 수 있을 것 같아요
(eslint-disable-next-line @typescript-eslint/no-explicit-any 사용하지 않을 수 있음)

```suggestion
    onError: (error) => {
    if(!isSeverError(error)) {
    return 
    }
    
      if (error.response.data.data.errorClassName === 'MISSION_RECORD_ALREADY_EXISTS_TODAY') {
        removeProgressMissionData();
        router.replace(ROUTER.HOME);
      }
    },
```